src/scss/layout/_title.scss

```diff
@@ -52,3 +52,16 @@
     }
   }
 }
+
+span.first {
+  display: block;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
```

Comment:
Why are we using a new font-family for this?

Reply:
My mistake, for some reason I thought I checked the browser computed fonts and this came up? Must have been some other issue